### PR TITLE
first pull request

### PR DESCRIPTION
**Pull request by Kidd254**
I applied nine changes to this branch such as creating html and css files, adding es linter configuration files, configuring linter, and modifying html, css, README, and gitignore files.

### CODE REVIEW

**Review by Kidd254 — COMMENTED** [review]
short and concise commit messages 👍 
correct git flow 👍 
correct use of semantic tags 👍
no linter errors 👍  
changes required ♻️ 
kindly apply the css box model to display content better

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <p>Select A Microverse Pair Programmer</p>
+                <select name="programmer" id="dropdown">
+                    <option value="Lawrence">Lawrence</option>
+                    <option value="Valentine">Valentine</option>
+                    <option value="John">John</option>
+                </select>
+            </section>
+            <section>
+                <fieldset class="form-group">
+                    <p>Select the Country</p>
```
Comment (Olamarx) [inline index.html:39]: - [ ] This is an example, you should also make your comment under the code or block of code. This is called an inline comment.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Kidd254, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Grammatically correct comments ✔️
- Used GitHub review feature ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kidd254** [comment]
@Olamarx  thank you for the positive review.. kindly describe to me how i can add a gif like yours, i think it's cool 😃

File: style.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#image {
+  max-width: 100%;
+  max-height: auto;
```
Comment (Kidd254) [inline style.css:3]: Apply the css box model so as to make the css file appear professional.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+}
+
+div#image {
```
Comment (Kidd254) [inline style.css:6]: apply the box model to make your css file appear professional

**Review by Kidd254 — COMMENTED** [review]
Nicely done 👋 
However, you could do better by applying the box model concept in your css file to make your css appear professional.
almost-there-gif-1

**Review by TadesseDev — APPROVED** [review]
### Hi @Kidd254 

### I'm Tadesse Alemayehu, your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve_happy


## Highlights
 - [x] You use GitHub inline comment feature 💯
 - [x]  You comments are clear and prcie. 👍🏾
 
## Required Changes ♻️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [Optional] include a `gif` image in your pull request summery, this would make your pull request friendly. I have added a Video description of how.

[Screencast from 03-10-2022 12:37:49 ከሰዓት.webm](https://user-images.githubusercontent.com/69077061/193547471-ead7ad28-cb8f-4495-942f-ebfa2dbfb70d.webm)


- [Optional] You are doing great @Kidd254 . The Only thing left now is to explain why you are requesting the changes. Example would be:-
- - > `Apply the css box model.` This would make your style professional.
This is only optional because it has not been mention to you by a previous code reviewer. I strongly suggest you rephrase your comments by adding explanation of why change is requested.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kidd254** [comment]
@Tadesse-Alemayehu thanks for the positive feedback. You've been of great help 👍 
thumbs-up-gif-funny-2

**Comment by TadesseDev** [comment]
### Good job @Kidd254 👌

**Pull request merged**